src/components/v2/Table/index.tsx

```diff
@@ -80,7 +88,7 @@ export function Table({ columns, data, title, minWidth, initialOrder }: ITablePr
 
         <TableBody>
           {rows.map(row => (
-            <TableRow key={uid(row)}>
+            <TableRow hover key={uid(row.length)} onClick={() => rowOnClick && rowOnClick(row)}>
```

Comment:
Unfortunately that doesn't work either, because the rows all have the same length, resulting in them having the same key:
<img width="240" alt="Screenshot 2022-04-08 at 08 48 09" src="https://user-images.githubusercontent.com/44675210/162390266-773e4adc-3140-4aa5-94ae-44c1174d4458.png">

A way to fix that can be to change the data model so that each row is an object that contains a `key` and the cells.
If you want to keep using an array for cells, you can simply change the type of table rows to something like that:
```typescript
type Row = {
  key: string;
  cells: ITableRowProps[];
}
```

Side note: `ITableRowProps` should probably have another name, since it's the type of a cell rather than a row.